src/SegmentedArray.chpl

```diff
@@ -196,7 +196,7 @@ module SegmentedArray {
       // NOTE: cannot compute lengths inside forall because agg.copy will
       // experience race condition with loop-private variable
       var right: [D] int, left: [D] int;
-      forall (r, l, idx) in zip(right, left, iv) with (var agg = new SrcAggregator(int)) {
+      forall (r, l, idx) in zip(right, left, iv) with (var agg = newSrcAggregator(int)) {
```

Comment:
did you mean to remove the space? where is the proc newSrcAggregator?

Reply:
Yeah meant to remove. `newSrcAggregator` and `newDstAggregator` are the wrappers that select between aggregation and unordered copy (based off comm={none,ugni,etc.})

Previously, this was always forcing aggregation even though that's not what you want for comm=none (and currently for ugni.) Everything else uses the wrappers, this was the one spot that didn't.

Reply:
Ok, sounds good.